### ISSUE TITLE
Emoji preview hidden when replying

### ISSUE THREAD

**Issue by vsego** [telegramdesktop/tdesktop#16667]
**To reproduce:**
1. Click "Reply" to some message.
2. Type `:foo`, where "foo" is the beginning of some emoji code (for example, `:jo` for `:joy:`).

**What happens**

The emoji popup is not shown:
![image](https://user-images.githubusercontent.com/8740710/126209184-393d7962-d5b8-493a-86d3-abf0c73400fa.png)

However, emojis do work. If you finish the code (for example, finish typing `:joy:`), you'll have it replaced by the 😂 emoji.

The preview is there, but hidden. If you click on the preview of the message you're replying to, i.e., the green area here:
![image](https://user-images.githubusercontent.com/8740710/126209451-3ca2105a-997b-4999-a449-e172bb3165fa.png)
the emoji popup will appear:
![image](https://user-images.githubusercontent.com/8740710/126209489-b822a8a8-5714-4df0-9f77-0b2472c05959.png)
It seems that the preview of the replied message is above the emoji popup (although, it's not clear why it drops behind when you click on it).

**Expected behaviour**

See the emoji popup when typing the emoji code, same as it appears when typing normal messages (i.e., non-reply ones).

**Technical notes**

It has been like this for a long time. The above screenshots were made under the current version (2.8.10), under xfce 4.14 / Fedora 31.

**Comment by ilya-fedin** [comment]
Can't reproduce on MATE with NixOS, works just fine after clicking reply

**Comment by ilya-fedin** [comment]
Also can't reproduce on Manjaro with Xfce 4.16

**Comment by ilya-fedin** [comment]
So looks like some reproduction steps are missing or the bug is in Xfce 4.14 and fixed in Xfce 4.16. More info is needed.

**Comment by ilya-fedin** [comment]
Oh, you have removed template and replaced with custom one
@Aokromes  can you close the issue?

**Comment by Aokromes** [comment]
@vsego you have downloaded telegram from desktop.telegram.org or from arch page?

**Comment by vsego** [comment]
@Aokromes, no idea. It was quite a while back and I forgot how I did it. Now I just click "Update" in it. How can I find out?

**Comment by ilya-fedin** [comment]
I think it's a bug in your old Xfce, please upgrade your system

**Comment by vsego** [comment]
@ilya-fedin, ok. Thank you. It's been that way across several minor versions of Xfce, so I assumed it was Telegram and not Xfce. A proper upgrade will probably have to wait a bit, as it requires a full Fedora upgrade. I'll close this now and reopen if it's still here under Xfce 4.16 after I upgrade.

Apologies for opening the issue in the wrong project and thank you for your help.

**Comment by ilya-fedin** [comment]
apparently old xfwm prevents Telegram to show the popup or something

**Comment by 23rd** [comment]
@vsego Can you tell me a certain place where this behavior occurs? Is it a regular conversation or perhaps a Discussion section?

**Comment by vsego** [comment]
@23rd, every chat I tried. With one person or with many (a group one). The above screenshots were done in Saved Messages.

It does feel like @ilya-fedin is right. I tried to make a video clip of the bug with Vokoscreen, and the bug wouldn't happen while its overlay was above Telegram, which suggests that the problem is outside of it.

**Comment by 23rd** [comment]
@vsego Okay, thanks.

**Comment by vsego** [comment]
I upgraded to Fedora 34 with Xfce 4.16. Interestingly, this didn't fix the problem. Later on, I realised that I was still running the old kernel (5.6.13-200, which came with Fedora 31) because the upgrade upgraded all the packages, but not Grub config. Once I fixed that and rebooted under kernel 5.12.15-300 (the current one for Fedora 34), the bug went away. I did no other changes to achieve this. Maybe something with the video driver. I recall the old one having issues and I had to turn something off with kernel loading, about a year ago, because the machine was crashing about once a week... forgot what it was... and I'm not sure if it's relevant here.

TL;DR: Fixed with Xfce 4.16, but it also required a newer kernel.
